add RCLL comments (#74)
mlsa-book/MLSA: book/meas_rules.qmd

The ERV interpretation makes reporting of scoring rules easier within and between experiments.
For example, say in experiment A we have $L|S = 0.004$ and $L|B = 0.006$, and in experiment B we have $L|S = 4$ and $L|B = 6$.
The sophisticated model may appear worse at first glance in experiment A (as the losses are very close) but when considering the ERV we see that the performance increase is identical (both $R_L = 33\%$), thus providing a clearer way to compare models.

## Conclusion

:::: {.callout-warning icon=false}

## Key takeaways

> * Scoring rules are a useful tool for measuring a model's overall predictive ability, taking into account concordance and discrimination.

Reviewer: concordance and calibration?

> * Scoring rules are a useful tool for measuring a model's overall predictive ability, taking into account concordance and discrimination.

Author: ```suggestion
* Scoring rules are a useful tool for measuring a model's overall predictive ability, taking into account calibration and discrimination.
```

* Strictly proper scoring rules allow models to be compared to one another, which is important when choosing models in a benchmark experiment.

> * The majority of scoring rules are *not* strictly proper, however experiments have shown that improper rules still provide useful and trustworthy results [@Sonabend2022b]

Reviewer: ```suggestion
* Many scoring rules for censored data are *not* strictly proper, however experiments suggest that improper rules still provide useful and trustworthy results [@Sonabend2022b]
```

::::

:::: {.callout-important icon=false}

## Limitations

* Scoring rules can be difficult to interpret but ERV representations can be a helpful way to overcome this.
* There is no consensus about which scoring rule to use and when so in practice multiple scoring rules may have to be reported in experiments to ensure transparency and fairness of results.

> * The majority of survival analysis models return distribution predictions, which are complex to derive density functions, $f(t)$, from [@Rindt2022], hence measures such as RCLL and SDLL can often not be easily used in practice.

Reviewer: ```suggestion
* For non- and semi-parametric survival models that return distribution predictions, estimates of $f(t)$ are not readily available and require approximations [@Rindt2022], hence measures such as RCLL and SDLL can often not be directly used in practice.
```

> * Any measure that relies on the density, $f(t)$, is difficult to use in practice. The majority of survival analysis models return discrete distribution predictions due to estimation of the baseline survival with the Kaplan-Meier estimator. Hence, the probability *mass* not *density* function is returned by these models and measures such as RCLL and SDLL can often not be used in practice. Direct estimation of $f(t)$ is usually not impossible and the likelihood is termed 'intractable' [@Rindt2022].

Author: ```suggestion
* The majority of survival analysis models return distribution predictions, which are complex to derive density functions, $f(t)$, from [@Rindt2022], hence measures such as RCLL and SDLL can often not be easily used in practice.
```
